.gitignore

```diff
@@ -22,6 +22,8 @@ share/python-wheels/
 *.egg-info/
 
 # Documentation
+docs/
+docs/*
```

Comment:
Why do you have `docs/` and `docs/*`? `docs/` already ignores the whole folder!

Reply:
Ok sounds good, will change to `docs/`